cmd/create_test.go

```diff
@@ -1,37 +1,30 @@
 package cmd_test
 
 import (
-	"io/ioutil"
-	"os"
-	"path"
 	"testing"
-
-	"github.com/commitdev/commit0/cmd"
-	"github.com/commitdev/commit0/internal/templator"
-	"github.com/commitdev/commit0/internal/util"
-	"github.com/gobuffalo/packr/v2"
 )
 
 func TestCreateWorks(t *testing.T) {
-	tmpdir, err := ioutil.TempDir("", "commit0-")
-	if err != nil {
-		t.Fatal(err)
-	}
+	// @TODO : Figure out a way to test this
```

Comment:
Shane has some ci integration tests to check if the templates populated under `/tests`

Reply:
Yeah, this test needs to be moved to a different level now because now this function triggers prompts.